package.json

```diff
@@ -68,6 +68,7 @@
     ]
   },
   "dependencies": {
+    "@internetarchive/fetch-handler-service": "^1.0.0",
```

Comment:
Is this only being used in `app-root.ts` and `MockFetchHandler`? If so, could you put it in `devDependencies` so it doesn't get included in the public package? `devDependencies` only get used during dev time whereas `dependencies` get included by consumers of this package.

Reply:
Nevermind.. I see it being used in `review-form.ts`. Missed that before!